src/dataland_qa_lab/dataland/data_provider.py

```diff
@@ -0,0 +1,49 @@
+from dataland_backend.models.extended_document_reference import ExtendedDocumentReference
+from dataland_backend.models.nuclear_and_gas_data import NuclearAndGasData
+
+
+class DataProvider:
+    """Provide certain data from Nuclear and Gas datasets."""
+
+    @classmethod
+    def get_yes_no_values_by_data(cls, data: NuclearAndGasData) -> list[str]:
+        """Get Yes/No values of given dataset."""
+        referenced_reports = data.general.general
+        sections = [
+            referenced_reports.nuclear_energy_related_activities_section426,
+            referenced_reports.nuclear_energy_related_activities_section427,
+            referenced_reports.nuclear_energy_related_activities_section428,
+            referenced_reports.fossil_gas_related_activities_section429,
```

Comment:
This list of datapoints is repeated many times throughout the codebase. Wouldn't it make sense to unify this behind a common abstraction? Additionally, I am not a big fan of returning lists where you require specific knowledge of the order of elements to derive any value from it. Instead, we could maybe consider returning a dict or something that retains structural information? What dou you think?